projects/laji/src/app/shared-modules/laji-form/document-form-header/document-form-header.component.ts

```diff
@@ -101,30 +101,10 @@ export class DocumentFormHeaderComponent implements OnInit, OnChanges, OnDestroy
     return moment(date).format('DD.MM.YYYY');
   }
 
-  getNamedPlaceHeader(): Observable<string>[] {
+  getNamedPlaceHeader(): string[] {
     if (!this.form || !this._namedPlace) {
       return [];
     }
-    const headerFields = this.form.options?.namedPlaceOptions?.headerFields || ['alternativeIDs', 'name', 'municipality'];
```

Comment:
@olzraiti Changes you asked are here. I'm leaving now, but if there is some problem, i will fix tomorrow morning

Reply:
Looks good, thanks! I'll wait for the feature demo to build, test and merge